move change text instructions into md file

Dedekind561 opened the pull request:
-----
[View rendered exercises/changing_text.md](https://github.com/CodeYourFuture/fundamentals-blockly/blob/add-changing-text-md/exercises/changing_text.md)

Dedekind561 commented:
I'm going to do each change in a small easy to review PR

gregdyke commented on exercises/changing_text.md:
```diff
@@ -0,0 +1,40 @@
+
+## Extra exercises
+
+1.  Can you update the contents of the other three `<u>` elements? If you can't think of some words, you can use the "get a random word" block from the Values menu
```
annotate "get a random word" with code (see the README)

gregdyke commented on exercises/changing_text.md:
```diff
@@ -0,0 +1,40 @@
+      
+2.  Inside this block, add a `find the element with id` block using the css selector for the id of the adjective `<u>` element
+      
+3.  Inside this block, add a `set the text content to` block 
```
include the quotes around the block name (see the readme)

gregdyke commented on exercises/changing_text.md:
```diff
@@ -0,0 +1,40 @@
+
+1.  Go to the html blocks and from there add an `at the start` block
+      
+2.  Inside this block, add a `find the element with id` block using the css selector for the id of the adjective `<u>` element
```
This is changed in the original html to remove "css selector"

gregdyke commented on exercises/changing_text.md:
```diff
@@ -0,0 +1,40 @@
+      
+3.  Inside this block, add a `set the text content to` block 
+
+4. Set the value of the text in this element to pink ( or any other colour you like )
```
I think using a color adjectives might make it confusing with the use of color attributes in other exercises. Maybe suggest words like "big, small, scary, angry"?

Dedekind561 replied:
yeah good point

gregdyke commented on index.html:
```diff
@@ -127,28 +127,9 @@ <h2>Modifying html dynamically: changing text</h2>
-                    <li>Inside this block, add a <span class="blockname">"find the element with id"</span> block using the id of the first &lt;u&gt; element (<code>noun1</code>)</li>
-                    <li>Inside this block, add a <span class="blockname">"set the text content to"</span> block</li>
-                    <li>Set the value for the text of this block: <code>mouse</code></li>
-                    <li id="exercise_change_text_1">Click "run" to check the content has changed.</li>
```
id needs keeping for tests, see the README

gregdyke commented on index.html:
```diff
@@ -127,28 +127,9 @@ <h2>Modifying html dynamically: changing text</h2>
-                    <li id="exercise_change_text_1">Click "run" to check the content has changed.</li>
-                </ol>
-            </li>
-            <li id="exercise_change_text_2">Can you modify the contents of the other three <code>&lt;u&gt;</code> elements? If you can't think of some words, you can use the <span class="blockname">"get a random word"</span> block from the Values menu</li>
```
id needs keeping for tests

gregdyke reviewed (changes requested):
I'd like the block names to be "machine readable", hence the possibly overkill
**"`at the start`" block.** Maybe the quotes are a bit much and **`text of block` block** is enough?

Dedekind561 commented:
Hi Greg, I've tried to address your feedback, a few things:

1. I don't think it is possible to add an id attribute to a bullet point in markdown so I reverted back to html 
2. Is there a bug with the test registering  - both of the tests seem to pass whenever I click the run button ( regardless of the blocks I use ) - this is in the `changing_text` exercise
3. I've reverted the text for block label from `"I am a block label"` to `I am a block label` - I agree I think the quotation marks are overkill
4. I tried styling some of the text inside one of the `li` elements using a span and a class of "blockname"; however, it didn't pick up the style from the `index.html`. Any ideas why it isn't picking up the style from `index.html` ?

gregdyke commented on exercises/changing_text.md:
```diff
@@ -0,0 +1,40 @@
+
+## Extra exercises
+
+<li id="exercise_change_text_2"><span class="test-checkbox"></span>Can you update the contents of the other three &lt;u&gt; elements? If you can't think of some words, you can use the <code>get a random word</code> block from the Values menu</li>
```
you don't need the id, only the span on its own

gregdyke commented on exercises/set_colours.md:
```diff
@@ -35,7 +35,7 @@ The colour for a banana is probably not easy to see against a white background.
 <ul style="background-color: azure;">
       <li style="color:rgb(233, 233, 22);background-color: darkgrey;">Banana</li>
-      <li>Orange</li>
+      <li>Orange</li> 
```
this file, no change :)

gregdyke reviewed (changes requested):
I'll have to have a look at the tests.
The CSS from the html is not propagated into the markdown (there's ways of passing the markdown a CSS file, but by default it's in a shadow dom that doens't inherit from the css of the rest of the page)

Dedekind561 commented:
Just updated :)

Dedekind561 commented:
@gregdyke Just a small nudge on this one - do you think it is ready to go ?

gregdyke reviewed (approved):
Looks good, thanks for the nudge. I still think "Extra exercises" suggests that this is an optional bonus, rather than "now do this", but we can iterate if need be

Dedekind561 commented:
Yeah no problem, can definitely update this during another iteration
